protonvpn_cli/utils.py

```diff
@@ -184,6 +189,30 @@ def is_connected():
     return True if ovpn_processes != [] else False
 
 
+def wait_for_network():
+    """Check if internet access is working"""
+
+    print("Waiting for connection...")
+    max_waiting_time = 20 * 60  # 20 Minutes
```

Comment:
This is quite a long timeout for waiting computer to boot. Wouldn't it be better to stick to e.g. 2 minutes by default? Also is one of the magic numbers that need to be made configurable (I guess in a next iteration?)

Reply:
I'll make it configurable with the environment variable `PVPN_WAIT` that's needed anyway to start it. Then e.g. `PVPN_WAIT=300` makes it wait for 5 minutes

Reply:
Added this as I've proposed with the latest commit